### PR TITLE
TW27065

### PR DESCRIPTION
**Pull request by vaibhavypatel**

### CODE REVIEW

**Review by jadenpuckett — CHANGES_REQUESTED** [review]
Should do a rebase instead of a merge branch. Merge branch puts an unnecessary commit in the history.

**Comment by Grumb1e** [comment]
I believe that is only the case if you do a rebase merge. If you do a squash merge it will collapse into one commit. You will just have to be careful with what name github gives the resulting commit.

**Review by jadenpuckett — APPROVED** [review]

**Comment by jadenpuckett** [comment]
Alright. As long as the name of the squashed commit message is the proper name and not the merge branch commit name.

File: plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XWidget.java [hunk 1]
```diff
@@ -99,6 +101,7 @@ public abstract class XWidget {
    private OseeImage oseeImage;
    private final Map<String, Object> parameters = new HashMap<String, Object>();
    protected ArtifactToken enumeratedArt = ArtifactToken.SENTINEL;
+   protected IUserGroupArtifactToken userGroupTok = UserGroupArtifactToken.SENTINEL;
```
Comment (dondunne) [inline plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XWidget.java:104]: remove "Tok" from field and methods

File: plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XWidget.java [hunk 2]
```diff
@@ -628,4 +631,12 @@ public void setEnumeratedArt(ArtifactToken enumeratedArt) {
       this.enumeratedArt = enumeratedArt;
    }
 
+   public IUserGroupArtifactToken getUserGroupTok() {
```
Comment (dondunne) [inline plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XWidget.java:634]: remove Tok

**Review by dondunne — APPROVED** [review]

**Pull request merged**